### DISCUSSION TITLE
Linac head simulation validation

### DISCUSSION THREAD

**Discussion by bcmohamed** [OpenTOPAS/OpenTOPAS#90] [Feature-Request]
Even I had repeated the simulation more than 20 times by modifying the beam source parameters and also for water phantom . The problem persists till know. I don't know where is the problem and how to fix it.

Please find in a picture of PDD distribution compared to measurements (red colour).

Thank you![topas.jpg](https://github.com/user-attachments/assets/9b3af559-9467-4e71-b854-45b910283494)

**Comment by LouisLechNissen** [comment]
Hi,

Could you provide a little more information and preferably parameter files?

Best regards,
Louis

**Reply by bcmohamed** [reply]
Thank you for your response 
Find the parameters of target, source and primary collimator

**Reply by LouisLechNissen** [reply]
I have a difficult time troubleshooting with solely this, however, things you could check for could be if the field size is identical in experiment and MC. You could also measure the energy spectrum (e.g. by a phasespace scorer) to see if matches your experimental spectrum if you have so.

**Reply by robin-physics** [reply]
Hello - why do you have such a large FWHM for the energy spread of the initial spead of electron energies? You have electrons of up to 10 MeV in the spectrum which you won't have on a linac.

The measured PDD looks about right with a relative dose of about 68%.

For scoring the dose in water phantom, what SSD and field size are you using?

**Reply by bcmohamed** [reply]
Hello Robin,
Thank you for your response.

The beam energy is 6MV and the SSD is 100 cm and the field size is 10x10 cm2.
Concerning the FWHM is 10% is 4.5 MeV I think.

**Reply by robin-physics** [reply]
Your beam spectrum is wrong - the energy spread on the initial electron beam will be much tighter than this. Having such a large variation of 4.5 MeV means you don't have a typical 6 MV beam.

I suggest to make it much less and you could start with what Ding used in his paper (2002) modelling a Varian linac for both 6 and 18 MV.
Here is screen shot from this paper but was using EGSnrc

![image](https://github.com/user-attachments/assets/20e4bac9-fe56-4a67-a26f-6fd11f567f75)

**Reply by robin-physics** [reply]
Mmmm - I just read the definition of the energybeamspread in the TOPAS docs. I would have a look here:

https://opentopas.readthedocs.io/en/latest/parameters/source/beam.html
